tests/system/load/test_llm.py

```diff
@@ -112,3 +112,44 @@ def test_llm_palm_score_params(llm_fine_tune_df_default_index):
         "evaluation_status",
     ]
     assert all(col in score_result_col for col in expected_col)
+
+
+@pytest.mark.flaky(retries=2)
```

Comment:
Unless really needed, add in small tests by default.

Reply:
Done.